CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ From the release 0.22.0, the changelog is maintained in the [GitHub releases](ht
 
 All notable changes to this project are documented in this file.
 
+### Fixed
+- Fix bug in `YarpRobotControl` (https://github.com/ami-iit/bipedal-locomotion-framework/pull/979)
```

Comment:
```suggestion
```

We are not updating the CHANGELOG anymore, see a few line before. Just provide a nice name for the PR (such as the one that you had) and then we can generate the changelog automatically for a release.